lte/gateway/c/core/oai/tasks/amf/amf_app_state_manager.cpp

```diff
@@ -134,23 +145,28 @@ void AmfNasStateManager::create_state() {
 
   // Initialize the local timers, which are non-persistent
   amf_nas_state_init_local_state();
+  OAILOG_FUNC_OUT(LOG_AMF_APP);
 }
 
 // Free the memory allocated to state pointer
 void AmfNasStateManager::free_state() {
+  OAILOG_FUNC_IN(LOG_AMF_APP);
   if (!state_cache_p) {
-    return;
+    OAILOG_FUNC_OUT(LOG_AMF_APP);
```

Comment:
Add log to indicate that state_cache_p is NULL

Reply:
Comment addressed.